pkg/acceptance/helpers/share_client.go

```diff
@@ -26,7 +26,6 @@ func (c *ShareClient) client() sdk.Shares {
 
 func (c *ShareClient) CreateShare(t *testing.T) (*sdk.Share, func()) {
 	t.Helper()
-	// TODO(SNOW-1058419): Try with identifier containing dot during identifiers rework
```

Comment:
Should we have also acceptance test for this (I've seen an integration one in grants_integration_test.go)?

Reply:
Added.